superset-frontend/src/SqlLab/components/TabbedSqlEditors.jsx

```diff
@@ -240,11 +256,10 @@ class TabbedSqlEditors extends React.PureComponent {
   }
 
   newQueryEditor() {
+    console.log(this.state.databaseIds);
     queryCount += 1;
     const activeQueryEditor = this.activeQueryEditor();
-    const firstDbId = Math.min(
-      ...Object.values(this.props.databases).map(database => database.id),
-    );
+    const firstDbId = Math.min(this.state.databaseIds);
```

Comment:
This won't work, right? Right now `getDatabasesIds` returns a list of databases.

Reply:
databaseIds is an array<numbers>